### PR TITLE
Add title and currency options in charts

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Hi @MaxGhenis - this pull request just contains a few small changes in the chart display function that add GBP support and a title argument in the function. Hope this helps, feedback is welcome.

### CODE REVIEW

File: microdf/charts.py [hunk 1]
```diff
@@ -17,6 +17,8 @@ def quantile_chg_plot(
     q=None,
     label1="Base",
     label2="Reform",
+    title="",
```
Comment (MaxGhenis) [inline microdf/charts.py:20]:
```suggestion
    title="Change in disposable income percentiles",
```

File: microdf/chart_utils.py [hunk 2]
```diff
@@ -11,3 +11,14 @@ def dollar_format(suffix=""):
     return mpl.ticker.FuncFormatter(
         lambda x, _: "$" + format(int(x), ",") + suffix
     )
+
+def gbp_format(suffix=""):
```
Comment (MaxGhenis) [inline microdf/chart_utils.py:15]: How about instead making a `currency_format(currency="USD", suffix="")` function, mapping the symbol from `currency`? Then `dollar_format` could call `currency_format` to reduce code duplication, and `quantile_chg_plot` could call `currency_format` directly?

File: microdf/chart_utils.py [hunk 3]
```diff
@@ -11,3 +11,14 @@ def dollar_format(suffix=""):
...
+    """GB Pound formatter for matplotlib/
+
+    :param suffix: Suffix to append, default is empty.
+    :returns: FuncFormatter.
+
+    """
+    return mpl.ticker.FuncFormatter(
+        lambda x, _: "£" + format(int(x), ",") + suffix
+    )
```
Comment (MaxGhenis) [inline microdf/chart_utils.py:24]: add newline

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Thanks for this enhancement, @nikhilwoodruff! Left a couple notes.

File: microdf/chart_utils.py (hunk 2 above)
Comment (nikhilwoodruff) [reply microdf/chart_utils.py]: Yeah that's a better idea, just added it.

**Pull request merged**